src/components/common/accordion/types/index.ts

```diff
@@ -0,0 +1,17 @@
+type HeaderProps = {
```

Comment:
store type of props for each component in it's file, also name all of them just Props